Libplanet/Action/ActionEvaluator.cs

```diff
@@ -389,10 +393,23 @@ ActionContext CreateActionContext(IAccountStateDelta prevStates, int randomSeed)
             // FIXME: Probably not the best place to have Validate().
             block.Validate(hashAlgorithm, currentTime);
 
-            return EvaluateTxs(
+            if (_evCache.TryGetValue(block.Hash, out IEnumerable<ActionEvaluation> cached))
+            {
+                _logger.Verbose(
+                    "_evCache hit. block.Hash: {blockHash}, # of cached: {count}",
+                    block.Hash,
+                    cached.Count()
+                );
+                return cached;
```

Comment:
According to coverage report, this code path is not tested.  It would be better if we test whether this method returns the same result for the same block hash.